### ISSUE TITLE
[Web] [Mobile] [Desktop] Can we introduce video support in the editor?

### ISSUE THREAD

**Issue by AcnoSaga** [singerdmx/flutter-quill#275]
My issue is about [Web]
My issue is about [Mobile]
My issue is about [Desktop]

Basically, along with the implementation of the image picker in the editor, can we have a video picker for the editor?

**Comment by singerdmx** [comment]
PR is welcome

**Comment by singerdmx** [comment]
Sample delta:
```
   {
      "insert": {
         "video": "https://www.youtube.com/watch?v=V4hgdKhIqtc&list=PLbhaS_83B97s78HsDTtplRTEhcFsqSqIK&index=1"
      }
   }
```

**Comment by singerdmx** [comment]
Also pay attention to https://github.com/singerdmx/flutter-quill/pull/295

**Comment by singerdmx** [comment]
Refer to https://github.com/singerdmx/flutter-quill/pull/298

**Comment by singerdmx** [comment]
https://stackoverflow.com/questions/65038430/youtube-video-in-video-player-package

**Comment by singerdmx** [comment]
What is left is to support Youtube video.
For other video we probably treat it as a link

**Comment by singerdmx** [comment]
Initial support is available in 1.7.0
